torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -1165,26 +1165,22 @@ struct MobileCodeImpl : CodeImpl {
   }
 
   void emitOperator(Node* node) override {
-    CodeImpl::emitOperator(node);
-    // const Operator& op = node->getOperator();
-    // if (op.hasOperation() && op.schema().is_vararg()) {
-    //   emitLoadInputs(node->inputs());
-    //   insertInstruction(OPN, operator_table_.size(), node->inputs().size());
-    // } else {
-    //   auto unique_op_name = op.schema().overload_name() != ""
-    //     ? op.schema().name() + "." + op.schema().overload_name()
-    //     : op.schema().name();
-    //   auto num_include = node->inputs().size();
-    //   // make sure we only do this for mobile code
-    //   if (op_to_num_specified_args_.find(unique_op_name) !=
-    //           op_to_num_specified_args_.end()) {
-    //     num_include = op_to_num_specified_args_[unique_op_name];
-    //   }
-    //   emitLoadInputs(node->inputs(), num_include);
-    //   insertInstruction(OP, operator_table_.size());
-    // }
-
-    // operator_table_.emplace_back(op.getOperation(node));
+    const Operator& op = node->getOperator();
+    if (op.hasOperation() && op.schema().is_vararg()) {
+      emitLoadInputs(node->inputs());
+      insertInstruction(OPN, operator_table_.size(), node->inputs().size());
+    } else {
+      auto unique_op_name = c10::toString(op.schema().operator_name());
+      auto num_include = node->inputs().size();
+      auto it = op_to_num_specified_args_.find(unique_op_name);
+      if (it != op_to_num_specified_args_.end()) {
+        num_include = it->second;
+      }
+      emitLoadInputs(node->inputs(), num_include);
```

Comment:
WHat is the use of num_include?

Reply:
That's the number of specified arguments.